ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentCraftPlanItemViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ElectronicObserver.Common;
+using ElectronicObserver.Data;
+using ElectronicObserver.Window.Tools.EquipmentUpgradePlanner.CostCalculation;
+using ElectronicObserverTypes;
+using ElectronicObserverTypes.Mocks;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner;
```

Comment:
Add a line of space between the namespace declaration and the class declaration.

Reply:
should be fine now